pt_BR.ini

```diff
@@ -183,12 +183,12 @@ Customize tilt = Configurar comandos por inclinação...
     To Calibrate = Para calibrar, mantenha o dispositivo sobre uma superfície plana e pressione "Calibrar".
     Calibrate D-Pad = Calibrar D-Pad
 
-OnScreen = Controles de toque na tela
-    Visibility = Visibilidad. #'Visibilidade' está longo e não cabe
+OnScreen = Controles na tela
+    Visibility = Visível? #'Visibilidade' está longo e não cabe
 Custom layout... = Editar layout de controles de toque
 Show Touch Pause Menu Button = Mostrar o botão do menu de pausa
 Disable D-Pad diagonals (4-way touch) = Desativar diagonais do D-Pad (4 direções)
-Button Opacity = Opacidade dos botões
+Button Opacity = Transparencia dos botões
```

Comment:
I think that opacity is the proper term, because 0 is the most transparent and 100 is the most opaque.